components/breadcrumb/src/overrides/link.js

```diff
@@ -33,7 +26,10 @@ const linkStyles = (_, { current }) => {
 	};
 };
 
-const linkAttributes = (_, { current, href }) => ({ href: current ? null : href });
+const linkAttributes = (_, { current, href }) => ({
+	href: href || '#0',
```

Comment:
Can we make this a default prop?
Also what is `#0`?

Reply:
`href="#0"` is a nice default href value to use, like the common `href="#"` but with the `0` it stops the scroll jump out of the box without requiring any js wizardry. 

`<a>` must have a `href` attribute in order to get assistive technology support (without having to muck around with tabindex etc).